### DISCUSSION TITLE
blog/2021-11-19_pfsense_opnsense_ipsec_cgnat/

### DISCUSSION THREAD

**Discussion by giscus** [Sieboldianus/du.nkel.dev#15] [Announcements]
# blog/2021-11-19_pfsense_opnsense_ipsec_cgnat/

A personal code notes blog

https://du.nkel.dev/blog/2021-11-19_pfsense_opnsense_ipsec_cgnat/

**Comment by Sieboldianus** [comment]
Comments welcome. If you do not want to allow Github API access for [giscus.app](https://giscus.app/), feel free to [directly add comments on Github](https://github.com/Sieboldianus/du.nkel.dev/discussions/15).

**Comment by jockyw2001** [comment]
Thanks for your notes! They were very helpful for me for migrating from Sophos UTM9 (end of life '26) to opnsense.
